ICAFIX/ReApplyFixMultiRunPipeline.sh

```diff
@@ -772,35 +774,35 @@ cd ..
 ## Rename some files (relative to the default names coded in fix_3_clean)
 ## ---------------------------------------------------------------------------
 
-# Remove any existing old versions of the cleaned data (normally they should be overwritten
-# in the renaming that follows, but this ensures that any old versions don't linger)
-/bin/rm -f ${concatfmri}_Atlas${RegString}_hp${hp}_clean.dtseries.nii
-/bin/rm -f ${concatfmri}_Atlas${RegString}_hp${hp}_clean_vn.dscalar.nii
+# # Remove any existing old versions of the cleaned data (normally they should be overwritten
+# # in the renaming that follows, but this ensures that any old versions don't linger)
+# /bin/rm -f ${concatfmri}_Atlas${RegString}_hp${hp}_clean.dtseries.nii
+# /bin/rm -f ${concatfmri}_Atlas${RegString}_hp${hp}_clean_vn.dscalar.nii
```

Comment:
What is the reason for commenting these out rather than updating the naming?  The intent seemed to be that if you run it and it fails, the outputs won't exist, but if it doesn't delete them first, then that won't be the case.

Reply:
If you update the naming to use the option, like the rest of the code, it won't delete the _clean files when a user runs with "clean_rclean".  I suppose if they forget to set this option, removing the _clean files is an unfortunate outcome, and the presence of the files isn't exactly the right way to check for errors, particularly after rerunning, so maybe not deleting things is the better option.

Reply:
If we leave these (and similar) lines in place and commented out, we should update the naming so it uses the new variable, so that uncommenting them won't be a disaster.  If we are sure we want to stop whatever the deletion was intended to do, then we can just remove the lines.

Reply:
I will update the naming with the new argument.